### PR TITLE
3669 url redirection

### LINKED ISSUE #3669: JAXRS:  URL redirection is intermittently incorrect
When accessing different URLs from multiple clients, the destination URL is intermittently returned with an incorrect IP address.

### PR DESCRIPTION
**Pull request by jim-krueger**
Fixes #3669

### CODE REVIEW

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java [hunk 1]
```diff
@@ -70,6 +70,7 @@
 import org.apache.cxf.transport.AbstractMultiplexDestination;
 import org.apache.cxf.transport.Assertor;
 import org.apache.cxf.transport.Conduit;
+import org.apache.cxf.transport.http.AbstractHTTPDestination.BackChannelConduit;
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java]: why is this necessary?

Comment (jim-krueger) [reply dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java]: It is an internal class that eclipse automatically adds for some reason... I'll remove it.

**Comment by jim-krueger** [comment]
#build
